### DISCUSSION TITLE
Delete action in DCAT apis, only returns 200.

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Informatievlaanderen/VSDS-LDESServer4J#1154] [Ideas]
[ldes/ldes-server:2.5.0-SNAPSHOT](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/issues/1058#)

![image](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/assets/15314807/67f30ddf-a724-4397-b145-b3e9e8b8fba1)
3 delete API calls in DCAT category only return 200 ok, even the dcat info is not there
![image](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/assets/15314807/ba365f5c-dd63-4ede-afd1-e1a9260e534c)
I think a 404 also needs to be return if the catelog is not there.

**Comment by Tomvbe** [comment]
I converted this to a discussion as I believe all our DELETE statements should return 200 and never a 404.
Here is an interesting article which explores the two options: https://medium.com/vena-engineering/to-404-or-not-to-404-fd5c9ca4f5df

**Comment by xdxxxdx** [comment]
>  If you are a purist or designing for a system where retries are both automatic and unknowable, then you may lean towards “always returning 200” to avoid any reliance on potentially inaccurate information. If you are a pragmatist or designing for a system where retries are controlled manually, you may prefer “differentiating between 404 vs 200” because having possibly inaccurate information is better than no information at all. 

I think DCAT API call is mainly triggered manually by the admin. there is no retry automatically happening?
